s4/commands/sync_command.py

```diff
@@ -102,7 +102,7 @@ def run(self):
                         client_2.get_uri(),
                     )
                     worker.sync(
-                        conflict_choice=self.args.conflicts, dry_run=self.args.dry_run
+                        conflict_choice=self.args.conflicts if self.args.conflicts else entry.get("conflicts", "ignore"), dry_run=self.args.dry_run
```

Comment:
you should be able to test this behaviour in https://github.com/MichaelAquilina/S4/blob/master/tests/test_sync.py#L191